BUILD

```diff
@@ -7,9 +7,10 @@ sh_binary(
         "setup.py",
         "//tensorflow_addons",
         "//tensorflow_addons/activations:activations_py",
-        "//tensorflow_addons/custom_ops:custom_ops_py",
+        "//tensorflow_addons/image:images_ops_py",
```

Comment:
How about removing all `_py` suffix? Perhaps  `images_ops`, `activations`, `layers` looks better. What do you think, Sean?

Reply:
Agree looks great.